justfile

```diff
@@ -222,13 +226,13 @@ ipython:
 ##########
 
 # Compile Sphinx documentation into HTML output
-sphinx-make: up wait-for-es wait-for-ing wait-for-web
+sphinx-make: _api-up
     just exec web sphinx-build -M html docs/ build/
 
 # Serve Sphinx documentation via a live-reload server
-sphinx-live port="3000": up wait-for-es wait-for-ing wait-for-web
-    just exec web sphinx-autobuild --host 0.0.0.0 --port {{ port }} docs/ build/html/
+sphinx-live: _api-up
+    just exec web sphinx-autobuild --host 0.0.0.0 --port 3000 docs/ build/html/
```

Comment:
Would we want to change this port to something in 502xx as well?

Reply:
The 3000 here refers to the port of the container (running inside `just exec`). It is correctly set to 3000.